lametro/templates/partials/board_report_statuses.html

```diff
@@ -0,0 +1,33 @@
+<table class="table">
+    <thead>
+        <tr>
+            <th>Status</th>
+            <th class="desktop-only">Description</th>
+        </tr>
+    </thead>
+    <tbody>
+    {% for status, obj in BILL_STATUS_DESCRIPTIONS.items %}
+        <tr>
+            <td>
+                <p class='nowrap'>
+                    {% if obj.search_term != None %}
+                    <a href='/search/?selected_facets=bill_type_exact%3A{{obj.search_term}}'>{{status}}</a>
```

Comment:
`bill_type_exact` will look in the Report Type facet for the specified value. You want the Status facet. Head over to the search on the live site and click one of the options under the Status facet to get the name of that facet from the URL.